tools/serverpod_cli/lib/src/runner/serverpod_command.dart

```diff
@@ -5,7 +5,7 @@ import 'package:serverpod_cli/src/logger/logger.dart';
 abstract class ServerpodCommand extends Command {
   @override
   void printUsage() {
-    log.info(usage, style: const LogStyle());
+    log.info(usage, style: const RawLog());
```

Comment:
`RawLog` is not a great name as it refers to a style the way it is used here (and other places). I would expect something like `PlainStyle()` or `LogStyle.plain`. `RawLog` make it sound like it is the logger itself.

Reply:
Hmm, not sure that I agree that Plain properly communicates what we want. Basically the log message is already formatted and should be printed as is, therefore my thought was that the log style would be "raw", that is, no formatting.

But maybe `PreFormatted`, `AsIs`, `NoStyle`, `Nothing` or `NoFormatting` better communicates this?